cmd/security-bootstrapper/Dockerfile

```diff
@@ -35,21 +35,22 @@ RUN make cmd/security-bootstrapper/security-bootstrapper
 FROM alpine:3.14
 
 LABEL license='SPDX-License-Identifier: Apache-2.0' \
-      copyright='Copyright (c) 2021 Intel Corporation'
+      copyright='Copyright (c) 2022 Intel Corporation'
 
 RUN apk add --update --no-cache dumb-init su-exec
 
-ENV SECURITY_INIT_DIR /edgex-init
-ARG BOOTSTRAP_REDIS_DIR=${SECURITY_INIT_DIR}/bootstrap-redis
+ENV SECURITY_INIT_STAGING_DIR /edgex-init-staging
+ARG BOOTSTRAP_REDIS_DIR=${SECURITY_INIT_STAGING_DIR}/bootstrap-redis
 
-RUN mkdir -p ${SECURITY_INIT_DIR} \
+RUN mkdir -p ${SECURITY_INIT_STAGING_DIR} \
+    && mkdir -p /edgex-init \
```

Comment:
`mkdir -p ${BOOTSTRAP_REDIS_DIR}` is the only thing needed; it will create any parent directories as needed; remove the others.